modules/photo/test/test_denoising.cuda.cpp

```diff
@@ -66,9 +66,13 @@ TEST(CUDA_BruteForceNonLocalMeans, Regression)
     cv::Mat gray;
     cv::cvtColor(bgr, gray, cv::COLOR_BGR2GRAY);
 
-    GpuMat dbgr, dgray;
+    cv::Mat gray16;
+    gray.convertTo(gray16, CV_16UC1);
```

Comment:
Why values range of processed image is NOT extended?

- 8U range is 0..255
- 16U range is 0..65535

It is really strange to run 16-bit processing on values from 8-bit range (you probably don't need this 16-bit processing).

Reply:
This is just for the test, I can add a 16-bit range image at '../gpu/denoising/nlm_denoised_lena_gray16' for example or scale this one.

Thanks